ports/gstreamer/portfile.cmake

```diff
@@ -1,83 +1,11 @@
-vcpkg_from_github(
```

Comment:
Does the new version merge these repos?

Reply:
Yes. They are all merged into `gstreamer` repo's `subprojects` folder. It seems like 1.19.2 was the preparation for the reorganization.